javascript/javascript3/week2/javascript3-homwork-week2/Find  Cool Api/main.js

```diff
@@ -0,0 +1,38 @@
+fetch("computers.json")
```

Comment:
Use async / await, not .then..catch. Additionally, see my code on how to import json in javascript. You don't need to use fetch. You can do it this way (for example, to read and display a json imported from a data.json file):

```html
<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <title>Display JSON Data</title>
  </head>
  <body>
    <h1>JSON Data</h1>
    <div id="json-data"></div>

    <script type="module">
      import jsonData from './data.json';

      const jsonDataDiv = document.getElementById('json-data');
      jsonDataDiv.innerHTML = JSON.stringify(jsonData);
    </script>
  </body>
</html>
```

Reply:
Okay I will use them